app/helpers/metadata_helper.rb

```diff
@@ -27,19 +34,34 @@ def metadata
           case action_name
           when :index
           else
-            {
-              title: @title
-            }
+            { title: @title }
           end
         when "tracks"
           case action_name
+          when "index"
+            { title: "Language Tracks" }
           when "show"
             {
-              title: "#{@track.title} on Exercism",
```

Comment:
What's the logic behind taking this out?

Reply:
This will be appended with `| Exercism` as followed in the SEO guidelines I posted above.

Reply:
Ah. I missed that method. Gotcha.